rln/rln.go

```diff
@@ -393,6 +394,62 @@ func (r *RLN) GetLeaf(index MembershipIndex) (IDCommitment, error) {
 	return result, nil
 }
 
+// GetMerkleProof returns the Merkle proof for the element at the specified index
+// The output should be parsed as: num_elements<8>|path_elements<var1>|num_indexes<8>|path_indexes<var2>
+// where num_elements indicate var1 array size and num_indexes indicate var2 array size.
+// Both num_elements and num_indexes shall be equal and match the tree depth.
+// A tree with depth 20 returns 676 bytes = 8 + 32 * 20 + 8 + 20 * 1
+// Proof elements are stored as little endian
+func (r *RLN) GetMerkleProof(index MembershipIndex) (MerkleProof, error) {
+	proofBytes, err := r.w.GetMerkleProof(index)
```

Comment:
Should we validate the length of proofBytes to be at least 8 bytes long so the line 416 does not fail?

Reply:
indeed
https://github.com/waku-org/go-zerokit-rln/pull/17/commits/4e7f9e1f306eae1d67f871bb76e07bb9cfa8e08e